fc/cli/task.py

```diff
@@ -17,8 +17,16 @@ def task():
 @click.option('--username')
 @click.argument('task_id')
 @click.argument('state')
-def move(username: str, task_id: str, state: str):
-    cli_library.echo('Transitioning task')
+@click.option('--resolution')
+@click.option('--comment')
+def move(username: str, task_id: str, state: str, resolution: str, comment: str):
+
+    the_resolution = resolution
+
+    if state in ['Resolved', 'Closed']:
+        if resolution is None:
+            the_resolution = cli_library.prompt('Please enter a resolution',
```

Comment:
What was your reasoning behind having the extra `the_resolution` variable in `task.py` in the FCLI PR?